src/components/TexterRequest.jsx

```diff
@@ -122,6 +133,20 @@ class TexterRequest extends React.Component {
       );
     }
 
+    if (
+      !this.userCanRequest(this.props.data.currentUser.memberships) &&
+      settings.showDoNotAssignMessage
+    ) {
+      return (
+        <Paper>
+          <div style={{ padding: "20px" }}>
+            <h3>Not allowed to request</h3>
```

Comment:
Copy suggestions we can workshop!
```suggestion
            <h3>Assignment Request Disabled</h3>
```